go.mod

```diff
@@ -117,6 +119,7 @@ require (
 	google.golang.org/grpc v1.19.0
 	gopkg.in/Shopify/sarama.v1 v1.18.0 // indirect
 	gopkg.in/cheggaaa/pb.v1 v1.0.25 // indirect
+	gopkg.in/mgo.v2 v2.0.0-20180705113604-9856a29383ce // indirect
```

Comment:
Where did the mongo dependency come from?

Reply:
it comes from simplereach/timeutils https://github.com/simplereach/timeutils/blob/master/time.go#L10

Reply:
Gross.

Reply:
https://github.com/araddon/dateparse/pull/85

Reply:
Let's see if it gets accepted.